vexim/adminuserchangesubmit.php

```diff
@@ -30,10 +30,15 @@
   }
   if (isset($_POST['on_forward'])) {
     $_POST['on_forward'] = 1;
-    if(!filter_var($_POST['forward'], FILTER_VALIDATE_EMAIL)) {
-      header ("Location: adminuser.php?invalidforward=".htmlentities($_POST['forward']));
-      die;
+    $forwardto=explode(",",$_POST['forward']);
```

Comment:
`$_POST['forwardmenu']` makes everything difficult. I thought about a JS solution, however, if you select one address, it is copied to the field, but what should it do when you select a different one, should it be added to a list, should it be overwritten? The latter would be the easiest to implement and covers probably 90% of all cases. I think, I'll go with that and we implement a more sophisticated solution in the next version.